[feat] better clarify invalid named layout references (#5912)
sveltejs/kit: documentation/docs/04-advanced-routing.md

<h1>I am inside +layout-foo</h1>
```

> > Named layout should only be referenced from `+page.svelte` files

Author: Just realized this isn't enough, it also applies to (named) layout files. I prefer to not be too wordy, here so I'll change this to
```suggestion
> Named layout should only be referenced from Svelte files
```

> > Only the Svelte file should reference the named layout, not its accompanying `+page.js` for example

Reviewer: How about this?

```suggestion
> Named layout should only be referenced from the `+page.svelte` file
```

Author: Oh yeah that's way better 👍
